sprints/sprint2_1.md

```diff
@@ -38,6 +38,6 @@ This sprint encompasses the following usages:
     - There shall be at least one `CONSTANTGRADIENT` and either at least one `CIRCULARARC` or at least one `PARABOLICARC` segments nested in the vertical layout (see `IfcAlignmentVerticalSegment.PredefinedType`).
 - [Object Nesting](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/object-nesting.htm)
     - There shall be at least one `IfcReferent` on `IfcAlignment` with non-empty `Name`.
-    - All `IfcReferent` shall be nested in any `IfcAlignment`.
+    - All `IfcReferent` shall be nested in any `IfcAlignment`. The `IfcRelNests` relationship shall only nest `IfcReferent` instances (see https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/issues/112).
```

Comment:
Are we sure about this? We also have to consider Alignment Layout.

Reply:
Alignment layout uses another instance of `IfcRelNests`. Perhaps we need to reword a bit?

Reply:
Just adding something like "The `IfcRelNests` relationship nesting `IfcReferent` occurrences shall only nest instances of `IfcReferent` (see https://github.com/bSI-InfraRoom/IFC-Tunnel-Deployment/issues/112)."

Reply:
Where is this adressed in the documentation?